pkg/graph/resolvers/system_intake_document.go

```diff
@@ -27,17 +27,17 @@ func GetSystemIntakeDocumentsByRequestID(ctx context.Context, id uuid.UUID) ([]*
 }
 
 // GetURLForSystemIntakeDocument queries S3 for a presigned URL that can be used to fetch the document with the given s3Key
-func GetURLForSystemIntakeDocument(ctx context.Context, store *storage.Store, s3Client *upload.S3Client, s3Key string) (string, error) {
+func GetURLForSystemIntakeDocument(ctx context.Context, store *storage.Store, s3Client *upload.S3Client, s3Key string) (*string, error) {
```

Comment:
Because I needed to allow for `null` document URLs in case the user is not allowed to view them. I modified the permissions code to not throw errors if the user can't view.